filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -937,8 +937,36 @@ void OpenGLDriver::framebufferTexture(backend::TargetBufferInfo const& binfo,
         attachmentTypeNotSupportedByMSRTT = true;
     }
 
-    if (rt->gl.samples <= 1 ||
-        (rt->gl.samples > 1 && t->samples > 1 && gl.features.multisample_texture)) {
+    // The following 4 techniques are different mechanisms to provide storage to the FBO.
+    // We'll use the first technique that is sufficient.
+    // nonMSAA:         MSAA was not requested, or we're rendering to an MS texture which
+    //                  doesn't require any special setup here
+    // useMSRTT:        use the EXT_multisampled_render_to_texture extension
+    // renderDirect:    texture is not sampleable, render directly into a renderbuffer
```

Comment:
So this is similar to `nonMSAA`, except here, this is supported by ES3.0 because the texture is not sampleable, so we can use a renderbuffer (which, well, is not a texture). But the point here is that there won't be any automatic resolve either.

Maybe it would make sense to refactor this so that the first case is just "noMSAA" and this case handles both sampleable and non sampleable MS textures, but in the case of sampleable MS textures it would "do nothing", i.e. revert to no MSAA.